engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -6939,6 +6954,12 @@ double demon_hunter_t::composite_player_multiplier( school_e school ) const
 {
   double m = player_t::composite_player_multiplier( school );
 
+  if ( set_bonuses.t30_vengeance_2pc->ok() &&
+       set_bonuses.t30_vengeance_2pc_buff->effectN( 1 ).has_common_school( school ) )
+  {
+    m *= 1.0 + buff.t30_vengeance_2pc->check() * set_bonuses.t30_vengeance_2pc_buff->effectN( 1 ).percent();
```

Comment:
I think you probably could simplify to `m *= 1.0 + buff.t30_vengeance_2pc->stack_value()`? Think you are loading the effect into the default value of the buff via the ->set_default_value_from_effect( 1 )?